### DISCUSSION TITLE
Thông tin

### DISCUSSION THREAD

**Discussion by KevinNitroG** [bigdargon/hostsVN#317] [Q&A] [answered]
Cho mình hỏi bên bạn với bên ABPVN có quan hệ hay hợp tác gì với nhau không? Mình mới biết đến sản phẩm của bạn. Để mình trải nghiệm xem sao :P

**Comment by bigdargon** [comment, marked as answer]
@KevinNitroG Chào bạn!

Mình và ABPVN là 2 dự án độc lập, ABPVN là dự án lọc qc bằng rule trên trình duyệt thông qua các tiện ích như uBlock, Adblock, Adguard… còn hostsVN là dự án lọc qc bằng tên miền thông qua cách thức lọc bằng DNS, file hosts của hệ điều hành, hay các trình phân giải khác.

Hiện tại, mình và ABPVN hợp tác với bộ hosts được cập nhật tự động từ github của mình, chi tiết ở đây https://abpvn.com/android/

Dự án của mình đang làm nguồn upstream cho các dự án khác như Steven Black’s hosts, Adguard DNS,… và các nhiều nguồn khác nữa. Dự án đang phát triển chặn qc/tracking ở VN và dự kiến sẽ mở rộng chặn thêm 1 số hạng mục khác nữa!
